src/utils/format-image-resizer-src/index.test.js

```diff
@@ -0,0 +1,57 @@
+import formatSrcset from ".";
+
+describe("format sourceset", () => {
+	it("without width or height params", () => {
+		expect(
+			formatSrcset("https://www.test.org/image.jpg", {
+				auth: "xyz123",
+				filter: 70,
+				smart: true,
+			})
+		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true");
+	});
+
+	it("with width and height included as resizer options", () => {
+		expect(
+			formatSrcset("https://www.test.org/image.jpg", {
+				auth: "xyz123",
+				filter: 70,
+				smart: true,
+				width: 100,
+				height: 100,
+			})
+		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true&width=100&height=100");
+	});
+
+	it("with width and height as explicit params, not as resizer options", () => {
+		expect(
+			formatSrcset(
+				"https://www.test.org/image.jpg",
+				{
+					auth: "xyz123",
+					filter: 70,
+					smart: true,
+				},
+				50,
+				50
+			)
+		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true&width=50&height=50");
+	});
+
+	it("with width and height explicit params overriding dimensions provided as resizer options", () => {
+		expect(
+			formatSrcset(
+				"https://www.test.org/image.jpg",
+				{
+					auth: "xyz123",
+					filter: 70,
+					smart: true,
+					width: 100,
+					height: 100,
+				},
+				50,
+				50
+			)
+		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true&width=50&height=50");
+	});
+});
```

Comment:
Just a rename of `formatSrcSet` to be consistent.

```suggestion
import formatSrc from ".";

describe("format sourceset", () => {
	it("without width or height params", () => {
		expect(
			formatSrc("https://www.test.org/image.jpg", {
				auth: "xyz123",
				filter: 70,
				smart: true,
			})
		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true");
	});

	it("with width and height included as resizer options", () => {
		expect(
			formatSrc("https://www.test.org/image.jpg", {
				auth: "xyz123",
				filter: 70,
				smart: true,
				width: 100,
				height: 100,
			})
		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true&width=100&height=100");
	});

	it("with width and height as explicit params, not as resizer options", () => {
		expect(
			formatSrc(
				"https://www.test.org/image.jpg",
				{
					auth: "xyz123",
					filter: 70,
					smart: true,
				},
				50,
				50
			)
		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true&width=50&height=50");
	});

	it("with width and height explicit params overriding dimensions provided as resizer options", () => {
		expect(
			formatSrc(
				"https://www.test.org/image.jpg",
				{
					auth: "xyz123",
					filter: 70,
					smart: true,
					width: 100,
					height: 100,
				},
				50,
				50
			)
		).toBe("https://www.test.org/image.jpg?auth=xyz123&filter=70&smart=true&width=50&height=50");
	});
});
```

Reply:
I dig consistency!